unstructured/partition/utils/constants.py

```diff
@@ -1,2 +1,3 @@
 SORT_MODE_XY_CUT = "xy-cut"
 SORT_MODE_BASIC = "basic"
+XY_CUT_BBOX_SHRINK_FACTOR = 0.9
```

Comment:
```suggestion
XY_CUT_BBOX_SHRINK_FACTOR = 0.9
```
please make this the default but make configurable via env var